Marlin/src/inc/Warnings.cpp

```diff
@@ -718,8 +718,8 @@
   #warning "Disabled CONFIGURATION_EMBEDDING because the target usually has less flash storage. Define FORCE_CONFIG_EMBED to override."
 #endif
 
-#if HAS_LCD_CONTRAST && LCD_CONTRAST_MIN >= LCD_CONTRAST_MAX
-  #warning "Contrast cannot be changed when LCD_CONTRAST_MIN >= LCD_CONTRAST_MAX."
+#if HAS_LCD_CONTRAST && LCD_CONTRAST_MIN >= LCD_CONTRAST_MAX && DISABLED(NO_LCD_CONTRAST_WARNING)
+  #warning "Contrast cannot be changed when LCD_CONTRAST_MIN >= LCD_CONTRAST_MAX. (Define NO_LCD_CONTRAST_WARNING to suppress this warning.)"
 #endif
```

Comment:
I don't understand how users would be getting to this particular contrast warning. These are all set in Conditionals_post.h. While they _could_ be overridden in configs, they aren't present there by default. This is warning of something someone went out of their way to add on top of the normal configs.

Since these aren't even user-defined values I'm wondering if this particular check should go the other way, and become an `#error`. If any of the default logic is causing MIN > MAX, then that should be fixed as a big rather than treated as a warning.

Reply:
> These are all set in Conditionals_post.h

This is true and it looks like the scenarios that where this warning used to trigger have been adjusted. We used to have displays with the same `LCD_CONTRAST_MIN` and `LCD_CONTRAST_MAX` values that would trigger the warning.

I'd rather allow this to be suppressible now or simply remove the warning completely.

Reply:
Adding it is fine. The warning was added for a reason at some point, and the debate over whether that remains valid can occur outside of this PR.

This adds no noise outside of the warnings file, so I don't see any issue with making any warning able to be disabled.